lib/core/src/Cardano/Wallet/DB/Sqlite/CheckpointsOld.hs

```diff
@@ -225,19 +234,20 @@ mkStoreCheckpoints wid =
     Database operations
 -------------------------------------------------------------------------------}
 selectAllCheckpoints
-    :: forall s. PersistState s
+    :: forall s. PersistAddressBook s
     => W.WalletId
     -> SqlPersistT IO [(W.Slot, W.Wallet s)]
 selectAllCheckpoints wid = do
     cps <- fmap entityVal <$> selectList
         [ CheckpointWalletId ==. wid ]
         [ Desc CheckpointSlot ]
-    fmap catMaybes $ forM cps $ \cp -> do
+    -- FIXME LATER during ADP-1043: Presence of these tables?
+    prologue <- fromJust <$> loadPrologue wid
```

Comment:
so here we have assumption that static part (aka Prologue) never changes ... we will need to be careful here when adding support for multi accounts... list of acctXPub, so a given accXPub, could stop to be static...just writing for future ourselves;-)

Reply:
Yes. Perhaps a good way to think about is that the "static part" is only "static" in the sense that it does not depend on information that discovered on the blockchain, but not "static" in the sense that it can never change. That's one of the reasons why I have called it "prologue" instead of "static". 😄